src/app/forms/templates/general/hgv-trl-body.template.ts

```diff
@@ -0,0 +1,52 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes } from '../../services/dynamic-form.types';
+
+export const hgvAndTrlBodyTemplate: FormNode = {
+  name: 'bodySection',
+  label: 'Body',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'make',
```

Comment:
Quite a lot of these fields are mandatory, need to add validators.

Reply:
Done